nawab.py

```diff
@@ -82,7 +86,7 @@ def main():
     thread.start()
     while thread.is_alive:
         #Initiate telegram bot
-        tg_bot_run(data, default_dir,level)
+        tg_bot_run(data, default_dir, level, args['retweet'], args['tg_bot'])
```

Comment:
this can be passed in an if condition if the retweet argument is passed put the param `args['retweet']`
else we don't need it